torchmdnet/datasets/ace.py

```diff
@@ -309,9 +317,7 @@ def get(self, idx):
         atoms = slice(self.idx_mm[idx], self.idx_mm[idx + 1])
         z = pt.tensor(self.z_mm[atoms], dtype=pt.long)
         pos = pt.tensor(self.pos_mm[atoms], dtype=pt.float32)
-        y = pt.tensor(self.y_mm[idx], dtype=pt.float32).view(
-            1, 1
-        )  # It would be better to use float64, but the trainer complaints
+        y = pt.tensor(self.y_mm[idx], dtype=pt.float64).view(1, 1)
```

Comment:
I would pass dtype as an argument to Ace here and store everything in the correct type. I do not see why store pos in float32 and y in float64.